go.mod

```diff
@@ -505,7 +505,7 @@ replace golang.org/x/time => github.com/cockroachdb/x-time v0.3.1-0.202305251236
 
 replace github.com/gogo/protobuf => github.com/cockroachdb/gogoproto v1.3.3-0.20241216150617-2358cdb156a1
 
-replace storj.io/drpc => github.com/cockroachdb/drpc v0.0.0-20250507084558-a793c5c40d3d
+replace storj.io/drpc => github.com/shubhamdhama/drpc v0.0.0-20250513152536-21683173691a
```

Comment:
Hmm, I'm uncertain about us moving from the `cockroachdb/drpc` fork to your personal fork. Is there a reason for this or is this an oversight?

Reply:
Oh, definitely not; we won't be moving out personal fork. I've left a comment here: https://github.com/cockroachdb/cockroach/pull/145195#issuecomment-2888975447 about merging the PRs from the fork first.

I had hoped to check this box before merging, but I realize now I should also add a "stacked on" remark, similar to my other PRs (e.g., https://github.com/cockroachdb/cockroach/pull/146926). I think I should also move this checkbox to the PR description itself.

Reply:
I've updated the PR description, I hope that should make it more explicit. I'm not sure if I should also put a "do-not-merge" as that may imply PR is not ready for review (maybe it's just me).